PCbuild/build.bat

```diff
@@ -123,6 +123,13 @@ if "%do_pgo%" EQU "true" if "%platf%" EQU "x64" (
     )
 )
 
+if "%UseDisableGil%" EQU "true" if "%UseTIER2%" NEQ "" (
```

Comment:
Strictly speaking this would be better as target in pythoncore.vcxproj (see the existing `_WarnAboutZlib` for example), since there's no _requirement_ to use the batch file, and it's very easy to bypass it. (The same goes for the PGO warning above, I don't remember when that was added.)

But it's not a big deal. It's relatively temporary code anyway

Reply:
Yeah, this will only be around for 3.14. I could just put an `#error` in the C source or whatever, but I figure failing early is best.

Besides, it's not like there's anything *wrong* with enabling both features. It just doesn't work the way you'd expect.